chemprop/v2/cli/train.py

```diff
@@ -156,7 +154,14 @@ def add_train_args(parser: ArgumentParser) -> ArgumentParser:
     mp_args.add_argument(
         "--activation",
         default="relu",
-        choices=['relu', 'leakyrelu', 'prelu', 'tanh', 'selu', 'elu'],  # TODO: should these be lowercase?
+        choices=[
+            "relu",
+            "leakyrelu",
+            "prelu",
+            "tanh",
+            "selu",
+            "elu",
+        ],  # TODO: should these be lowercase?
         help="activation function in message passing/FFN layers",
```

Comment:
activations are controlled by the `chemprop.v2.models.utils.Activation` enum

Reply:
so this means all lowercase is correct, right? or does something need to change?

Reply:
You can replace the choices with

```
list(Activation.keys())
```

to get all available ones.

Reply:
yeah just change this to
```python
choices=Activation.keys()
type=lambda x: x.lower()
```
The `type` arg is just a lambda that automatically puts all strings into lowercase so the input is case-insensitive.